.gitignore

```diff
@@ -27,6 +27,7 @@ docs/schema-loc-*.rst
 eclipse/
 es/elasticsearch-*
 es/es-dashboards/kibana-*
+es/es-dashboards/data/nodes/
```

Comment:
This folder was created when running following docker compose instructions for running ES.